content/en/tracing/version_tracking/_index.md

```diff
@@ -5,15 +5,88 @@ description: 'Use Datadog to track version during and after deployments'
+- Error rate as a percentage of total requests.
+
+**Note:** The 'Error Types' column is not a guarantee the version introduced these errors, but rather an indication that this error was not seen in traces from the previous version.  Based on how you are performing deployments, it can be a great way to begin investigating errors.
+
+Additionally, fields can be added or removed from this overview table and your selections will be saved.  The additional fields are:
+
+- Endpoints that are active in a version that were not in the previous version.
+- Time active, showing the length of time from the first trace to the last trace sent to Datadog for that version.
+- Total number of Requests.
+- Total number of Errors.
+- Latency measured by p50, p75, p90, p95, p99, or max.
+
+{{< img src="tracing/version_tracking/VersionComparison.png" alt="Versions on the Service Page"  style="width:100%;">}}
+
+**Note:** The `version` section of of the service page will only appear if there is more than one version reporting in the selected time interval at the top of the page.
+
+## Version Comparison
+
+Clicking on any version row in the Version Summary table will open a detailed version comparison page, allowing you to compare two specific version of the same service.  By default, the selected version will be compared to the immediately previous version but this can be changed to compare any two versions within the past 30 days.
+
+There are three main components of version comparison:
+
+- [Comparison Graphs](#comparison-graphs) for a visualization of requests and errors to services, useful for watching various types of [deployments](#version-deployment-strategies).
+- [Error Comparison](#error-comparison) to see errors that may have been introduced or solved by a deployment.
+- [Endpoint Comparison](#endpoint-comparison) to see how endpoint latency and error rates are performing in different versions of the service.
+
+### Comparison Graphs
+
+Requests and Error graphs are available in this section, much like the graphs on the Service Page. These graphs highlight the selected versions for comparison but leave all other versions in gray for additional context. These allow for a quick overview of a deployment rollout or the identification of a spike in error rates.
+
+{{< img src="tracing/version_tracking/ComparisonGraphs.png" alt="Version Comparison Graphs" style="width:100%;">}}
+
+### Error Comparison
+
+This section lists differences in error types detected for each version, highlighting error types appearing only in the source version for easy troubleshooting, error types no longer appearing there for easy validation of fixes and error types active in both.  From this table, you can pivot immediately into live or historical traces corresponding to the selected error for further investigation.
+
+**Note:** This is based on observed error types and if one error type is rare it might be represented as no longer appearing just because it has not been seen yet.
```

Comment:
```suggestion
**Note:** Error comparison is based on _observed_ error types. If an error type is rare, it might be listed as no longer appearing only because it has not been seen _yet_.
```